glide.yaml

```diff
@@ -0,0 +1,21 @@
+package: github.com/Azure/azure-sdk-for-go
+import:
+- package: github.com/Azure/azure-sdk-for-go
+  subpackages:
+  - /arm/examples/helpers
+  - arm/storage
+  - core/http
+  - core/tls
+  - management
+  - management/location
+  - management/virtualmachine
+  - management/virtualmachinedisk
```

Comment:
any ideas why this list is incomplete? (I'm not familiar to glide, it well may be because integration tests depend on these)